libs/langchain/langchain/embeddings/vertexai.py

```diff
@@ -6,11 +7,14 @@
 from langchain.llms.vertexai import _VertexAICommon
 from langchain.utilities.vertexai import raise_vertex_import_error
 
+logger = logging.getLogger(__name__)
+
 
 class VertexAIEmbeddings(_VertexAICommon, Embeddings):
     """Google Cloud VertexAI embedding models."""
 
     model_name: str = "textembedding-gecko"
+    show_progress_bar: bool = False
```

Comment:
@eyurtsev @baskaryan  Is it possible to add it to the base Embeddings? It looks like a useful generic capability.

Reply:
It would be nice, but the base Embeddings class is just an interface. And adding the attribute doesn't necessarily add the capability itself, because then you need to implement it on the embed_documents function